hack/ginkgo-e2e.sh

```diff
@@ -169,15 +169,21 @@ elif [[ "${E2E_TEST_DEBUG_TOOL}" == "gdb" ]]; then
   program=("gdb")
 fi
 
+debug_opt=()
```

Comment:
do we really need to add this complexity?
I prefer to remove it entirely, for someone that has to deal with ginkgo problems we should assume already checked all the open ginkgo issues and know exactly what to do.

Also, there is no much difference between uncommenting and adding the line directly, and doesn't seem to scale well if we have to comment all ginkgo flags here that can help debugging

Reply:
I was undecided myself about this. One advantage of this `debug_opt` is that one can comment out individual options. This cannot be done inside the full invocation below.

But I agree, this doesn't add a whole lot of additional value for the complexity.

Reply:
okay, I will update to just remove it.